package.json

```diff
@@ -38,7 +38,8 @@
     "contributors": "*",
     "eslint": "*",
     "mocha": "*",
-    "rewire": "*"
+    "rewire": "*",
+    "tmp": "^0.0.33"
```

Comment:
Also, please strip the version of this dependency. Odds are we almost always want the latest versions for DevDeps, and if it breaks the build it'll get caught by greenkeeper.

Reply:
ok -- haven't run into this approach before but stripping for consistency with the status quo

Reply:
The only people it'll inconvenience if a newer version of a dep breaks something is the developers, right? So why make them manually bump versions and test each time a dev dep is released? Especially when we've got tools like greenkeeper that can automatically do that for us....

Reply:
If you look [here](https://github.com/msimerson/maxmind-geolite-mirror/branches), you'll see the branches that greenkeeper automatically creates whenever a dependency releases a new version. If the tests fail, then it opens an issue.

Reply:
Yeah, this makes sense. Loose version requirements give me the willies in general but in this context I agree that there's no security issue and the potential for breakage in a dev environment is really just a matter of preference; happy to defer.